jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/importer/parser/BsmFileParserTest.java

```diff
@@ -44,7 +47,7 @@
    @Test
    public void testStepsAlreadyDone() {
 
-      ParserStatus expectedStatus = ParserStatus.COMPLETE;
+      ParserStatus expectedStatus = ParserStatus.INIT;
```

Comment:
This test is meant to verify that the parser returns COMPLETE status when all the steps are complete. Does the parser now change its status to INIT after completing?